src/components/DataTable/dataTable.module.scss

```
@@ -0,0 +1,207 @@
+    .stacked .table tbody tr,
+    .stacked .table tbody td {
+        display: block;
+        width: 100%;
+    }
+
+    .stacked .table thead {
+        position: absolute;
+        overflow: hidden;
+        width: 1px;
+        height: 1px;
+        clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)